tests/test_predict.py

```diff
@@ -38,19 +38,14 @@ def step(self, action):
 @pytest.mark.parametrize("model_class", MODEL_LIST)
 def test_auto_wrap(model_class):
     # test auto wrapping of env into a VecEnv
```

Comment:
Bit odd to have two comments on different subjects back-to-back; maybe turn this comment into a docstring given it's about what the function does rather than its implementation anyway?